experimental/framework/fluid-static/src/containerCode.ts

```diff
@@ -87,11 +97,15 @@ export class RootDataObject
         });
 
         await Promise.all(initialObjectsP);
+
+        const lastEditedFluidObject = await LastEditedTrackerDataObject.getFactory().createChildInstance(this.context);
+        this.root.set(this.lastEditedFluidObjectId, lastEditedFluidObject.handle);
```

Comment:
We're currently trying to avoid built-in data objects in the core `ContainerRuntime` because (1) it's an unhelpful cost for customers who don't use them (e.g. if a customer doesn't care about LastEdited here, it's bundle size, op traffic, and storage use they don't benefit from) and also (2) it imposes a back-compat constraint on us to manage (e.g. suppose we come up with a better, but data-incompatible version of the last-edited tracker - making the change-over would break all existing customers).

Since the `FluidContainer` represents a hopeful future API surface for the `Container`, I think it would probably be best to keep this above the `FluidContainer` by forcing the caller to add it to the registry themselves, instantiate, etc. for similar reasons.  I think it could definitely go into a "framework" layer though.

Reply:
Good point! I've removed the default inclusion of the last-edited-experimental object from the container. Instead, like you mentioned, developers can choose to opt-in to this feature, and perhaps we can include some tutorials to show how last-edited can be added to the initial containerSchema passed in and be fetched back by the developers themselves, instead of wrapping that functionality into the rootDataObject